### PR TITLE
Feat: Add section on HC introspection commands & add page to scaffold and run hApps

### PR DESCRIPTION
**Pull request by jost-s**
added a section on commands to get versioning info about Holochain:
- `hn-introspect`
- `holochain --build-info`

added a new page on how to scaffold a hApp using
- `hn-init`
- `holochain-create`
and on how to run a packaged hApp with the Holochain Launcher.

@pdaoust @steveeJ I'd love some feedback on these changes from you guys.

Here's the preview https://deploy-preview-317--fervent-tesla-2a3da2.netlify.app/

### CODE REVIEW

File: src/happ-setup.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+---
+hide:
+  - toc
+---
+
+## Scaffolding a new hApp
+
+Commands that serve to set up a new Holochain app (hApp) are at your fingertips. There are two ways in which you can create a new hApp: generate an empty project or use a GUI to describe your app's structure and then generate it. **All of the following commands need to be run from a Holochain nix-shell.**
```
Comment (steveeJ) [inline src/happ-setup.md:8]: do we assume that the reader knows how to set up holonix? how abouta link to the section where that's described?

File: src/happ-setup.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+### Create an empty project
+
+The command to set up a new empty project is
```
Comment (steveeJ) [inline src/happ-setup.md:12]: do we want to give some of the inner workings away and explain that this command is merely a wrapper around the scaffolding tool? also that the arguments are passed to that?

Comment (steveeJ) [reply src/happ-setup.md:12]: see https://github.com/holochain/holonix/blob/507049ba661aa0113f24e03d0f8307caaa3124a9/scaffolding/default.nix#L8

File: src/happ-setup.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+    └── web-happ.yaml
+
+20 directories, 30 files
+```
+
+You can enter the Holochain nix-shell and immediately start developing your hApp.
+
+### Create a new hApp using the GUI
+
+If you already know the data structure of your hApp or prefer a visual way of defining the hApp's entries, you can create the new hApp using a GUI.
```
Comment (steveeJ) [inline src/happ-setup.md:90]: for my own understanding: how does "already knowing the structure of your hApp" benefit running `holochain-create` vs `holochain-create init` (a.k.a. `hn-init`)?

File: src/happ-setup.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+```
+
+The resulting hApp is laid out as printed above. A web app is executed and opens in the default web browser. It provides controls to add DNAs and Zomes, and to add and configure
+field name and types of your hApp's entry definitions. Further you can choose between different templates for the hApp's UI.
+
+When you're done setting up the hApp in this graphical way, all files will be produced in the folder "my-app".
+
+## Running a deployed hApp
+
+At some stage in the app development you'll want to deploy your hApp for others to use it. If you've created the hApp with the scaffolding tools, all you need to do to package is is run
```
Comment (steveeJ) [inline src/happ-setup.md:103]:
```suggestion
At some stage in the app development you'll want to deploy your hApp for others to use it. If you've created the hApp with the scaffolding tools, all you need to do to package is run
```

File: src/install.md [hunk 5]
```diff
@@ -243,7 +243,54 @@ The next time you enter your hApp development environment using `nix-shell`, the
 
 > Keep in mind that the Holonix repo includes Nix configurations for the last ~ 5 versions of Holochain. That means that if you keep updating its revision using `niv`, you will have to augment the Holochain version id in `default.nix` sooner or later too.
 
-### Advanced usage
+### Holochain inspection commands
+
+Built into Holochain are a few commands that give insight about versions of Holochain components.
```
Comment (steveeJ) [inline src/install.md:248]:
```suggestion
Built into Holochain and holonix are a few commands that give insight about versions of Holochain components.
```

**Review by steveeJ — COMMENTED** [review]
thanks for writing this up :raised_hands: 

i added some questions that point to areas the text could be extended

File: src/happ-setup.md (hunk 3 above)
Comment (jost-s) [reply src/happ-setup.md:90]: If you know the structure of your hApp, you can use the GUI as a visual way of defining data entries as opposed to typing it out and copy/pasting it for all entries. If you don't know the structure, then the GUI won't be of use I think.

File: src/happ-setup.md (hunk 1 above)
Comment (jost-s) [reply src/happ-setup.md:8]: yes, sure

File: src/happ-setup.md (hunk 2 above)
Comment (jost-s) [reply src/happ-setup.md:17]: I can mention that it's a wrapper, yes. I don't see a point to talk about arguments, because there are none documented.

**Comment by jost-s** [comment]
@steveeJ I appreciate your feedback! All changes made, how about now?

**Review by steveeJ — APPROVED** [review]
thanks!

**Pull request merged**